src/py42/clients/cases.py

```diff
@@ -0,0 +1,112 @@
+class CasesClient(object):
+    """A client to expose cases API.
+
+    `Rest documenation <https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases>`__
```

Comment:
Wait we should not link to Core Int. Is there one in `console.us.cloud` we can use?

Reply:
Once I get access I will verify and update the same.